src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
+@import "./utils/reset.scss";
+@import "./utils/variables";
+@import "./utils/mixins";
+
 body {
-  margin: 0;
+  font-family: "Roboto", sans-serif;
 }
```

Comment:
```suggestion

```

Reply:
It's better to move your body styles to a separate file to keep your main file clean and clear